DOS/sso/internal/storage/lru.go

```diff
@@ -94,3 +96,50 @@ func setRedisClientUp(rcli **redis.Client) error {
 
 	return err
 }
+
+// Endorses the roles to a set identifier
+func EndorseAuthorites(asetID string, authorities map[string]interface{}) error {
```

Comment:
This function is not still in use. (But it works for sure)

It'll be applied in forthcoming pull request regarding managing authorities through redis

Reply:
me late cacahuate

Reply:
No quiero pecar de exagerado jaja, pero y si al recorrer el mapa vas concatenando los permisos en un string? Así te evitas el slice y la llamada a strings.join()

Reply:
pero no pasa nada, luego me obsesiono viendo alternativas

Reply:
El detalle seria definir, que no le agregue el espacio a el ultimo elemento iterado